packages/compass-e2e-tests/index.js

```diff
@@ -35,7 +35,10 @@ async function main() {
     mocha.addFile(path.join(__dirname, testPath));
   });
 
-  mocha.run((failures) => (process.exitCode = failures ? 1 : 0));
+  mocha.run((failures) => {
+    process.exitCode = failures ? 1 : 0;
+    process.exit();
```

Comment:
```suggestion
    // TODO: Figure out why tests don’t exit without this
    process.exit();
```

to communicate that this is not the ideal state and might hide bugs (even if we may not get around to actually doing this)?

Reply:
Good suggestion. Probably is hiding a bug or leaky thing. I'll add the comment and spend a bit more time looking into what could be causing it.